.github/workflows/fastly-edge-assignments.yml

```diff
@@ -0,0 +1,66 @@
+name: Fastly Edge Assignments
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types: [published]
```

Comment:
Releases have a commit on `main`, so having a `release` trigger here will just execute the same workflow twice.

It's also going to be executed on *all* releases (core, rust, ruby, python sdks)

Suggest dropping this trigger:
```suggestion
```